### PR TITLE
Goblin roles

### PR DESCRIPTION
**Pull request by panvxv**
## About The Pull Request
Adding goblin roles, King, Cook, Guard and Smith, all work in progress.
Made new eyes for playable goblins too, need of sprite overlay as I want yellow glowing eyes you can see in darkness (just like rauces and honey spiders have)

## Why It's Good For The Game
GOBO!!

### CODE REVIEW

**Comment by zakanater19** [comment]
Do they spawn in the goblin dungeon? because it seems like you are adding goblins into the actual town?

**Comment by panvxv** [comment]
> Do they spawn in the goblin dungeon? because it seems like you are adding goblins into the actual town?

They will spawn in goblin kingdom, it is WIP. I am working on it right now.

**Comment by Real-MAGNUM** [comment]
All of these need merging into a single job, except for king.

File: code/modules/surgery/organs/eyes.dm [hunk 1]
```diff
@@ -145,6 +145,12 @@
 	see_in_dark = 3
 	lighting_alpha = LIGHTING_PLANE_ALPHA_LESSER_NV_TRAIT
 
+/obj/item/organ/eyes/goblin
+	name = "goblin eyes"
+	desc = ""
+	see_in_dark = 15
```
Comment (BlueMemesauce) [inline code/modules/surgery/organs/eyes.dm:151]: max is 8 no need for this
```suggestion

```

Comment (BlueMemesauce) [inline code/modules/surgery/organs/eyes.dm:151]: 8 is the max
```suggestion
	see_in_dark = 8
```

**Comment by YosemiteYam** [comment]
There shouldnt be more than ten of them otherwise we're going from "minor faction" to "theres a major goblin kingdom on this small ass island that no one addressed for some reason".

And make it so that the slots do not automatically reopen, that shit will just induce goblins to be like respawnable bandits+ with their own fort basically. 

And make their base shittier/gear non existent, stone weapons and caves. Make them actually work and steal for everything else, like an evil version of the dwarf-fort gimick players who build everything from scratch.

**Comment by Xkallubar** [comment]
> There shouldnt be more than ten of them otherwise we're going from "minor faction" to "theres a major goblin kingdom on this small ass island that no one addressed for some reason".
> 
> And make it so that the slots do not automatically reopen, that shit will just induce goblins to be like respawnable bandits+ with their own fort basically.
> 
> And make their base shittier/gear non existent, stone weapons and caves. Make them actually work and steal for everything else, like an evil version of the dwarf-fort gimick players who build everything from scratch.

I'd rather the Goblins have some method of creating more goblin slots, perhaps something ashwalker-esque where the gobbos sacrifice corpses to open more gobbo slots. Otherwise the goblins will probably die out pretty easily due to how easy it is to die for dumb reasons.

**Comment by Xkallubar** [comment]
Can we get goblin shamans that have a heal, and maybe fireball and some poison or curse spells?

**Comment by Hmeister-real** [comment]
Gobo Head Spike - Put heads on of anyone who isn't a gobo or a skeleton, adds a new basic gobo spawn. Craftable by just goblins and limited to 3 heads per spike, maybe make them cost 2 twigs and then something hard to get like a steel ingot.

**Comment by zakanater19** [comment]
> Gobo Head Spike - Put heads on of anyone who isn't a gobo or a skeleton, adds a new basic gobo spawn. Craftable by just goblins and limited to 3 heads per spike, maybe make them cost 2 twigs and then something hard to get like a steel ingot. map a few around the gobo castle entrance

I think we should focus on just getting this game ready before bloat stuff

File: code/modules/surgery/organs/eyes.dm (hunk 1 above)
Comment (panvxv) [reply code/modules/surgery/organs/eyes.dm:151]: nah, when looking with shift+right click we can see up to 15 tiles

**Comment by erik30015** [comment]
hey uh I think that not setting a max PQ might end up fucking it up like the round it was merged

**Pull request merged**